galite-tests/src/test/kotlin/org/kopi/galite/demo/StoreApplication.kt

```diff
@@ -0,0 +1,434 @@
+  val rate = integer("TAX RATE IN %%")
+
+  override val primaryKey = PrimaryKey(idTaxe)
+}
+
+@SpringBootApplication
+open class StoreApplication : SpringBootServletInitializer()
+
+fun main(args: Array<String>) {
+  connectToDatabase()
+  DBSchemaTest.reset()
+  initDatabase()
+  initModules()
+  initUserRights()
+  addClients()
+  addTaxRule()
+  addProducts()
+  addFourns()
+  addStock()
+  addCmds()
+  addBills()
+  addBillPrdt()
+  runApplication<StoreApplication>(*args)
+}
+
+/**
+ * Initialises the database with creating the necessary tables and creates users.
+ */
+fun initDatabase(user: String = DBSchemaTest.connectedUser) {
+  transaction {
+    DBSchemaTest.createDBSchemaTables()
+    DBSchemaTest.insertIntoUsers(user, "administrator")
+    createStoreTables()
+  }
+}
+
+
+fun connectToDatabase(url: String = DBSchemaTest.testURL,
+                      driver: String = DBSchemaTest.testDriver,
+                      user: String = DBSchemaTest.testUser,
+                      password: String = DBSchemaTest.testPassword) {
+  Database.connect(url, driver = driver, user = user, password = password)
+  DBSchemaTest.connectedUser = user
+}
+
+/**
+ * Creates DBSchema tables
+ */
+fun createStoreTables() {
+  list_Of_StoreTables.forEach { table ->
+    SchemaUtils.create(table)
+  }
+}
+
+val list_Of_StoreTables = listOf(Client, Product, Stock, Provider,
+                                 Bill, TaxRule, Command, BillProduct)
+
+fun initModules() {
+  transaction {
+    DBSchemaTest.insertIntoModule("2000", "org/kopi/galite/test/Menu", 10)
+    DBSchemaTest.insertIntoModule("1000", "org/kopi/galite/test/Menu", 10, "2000")
+    DBSchemaTest.insertIntoModule("2009", "org/kopi/galite/test/Menu", 90, "1000", FormSample::class)
+    DBSchemaTest.insertIntoModule("2010", "org/kopi/galite/test/Menu", 90, "1000", FormWithFields::class)
+  }
+}
+
+fun initUserRights(user: String = DBSchemaTest.connectedUser) {
+  transaction {
+    DBSchemaTest.insertIntoUserRights(user, "2000", true)
+    DBSchemaTest.insertIntoUserRights(user, "1000", true)
+    DBSchemaTest.insertIntoUserRights(user, "2009", true)
+    DBSchemaTest.insertIntoUserRights(user, "2010", true)
+  }
+}
+
+
+fun addClients() {
+  transaction {
+    Client.insert {
+      it[idClt] = 0
+      it[nameClt] = "Salah"
+      it[fstnameClt] = "Mohamed"
+      it[addressClt] = "10,Rue du Lac"
+      it[cityClt] = "Megrine"
+      it[postalCodeClt] = 2001
+      it[ageClt] = 40
+    }
+    Client.insert {
+      it[idClt] = 1
+      it[nameClt] = "Guesmi"
+      it[fstnameClt] = "Khaled"
+      it[addressClt] = "14,Rue Mongi Slim"
+      it[cityClt] = "Tunis"
+      it[postalCodeClt] = 6000
+      it[ageClt] = 35
+    }
+    Client.insert {
+      it[idClt] = 2
+      it[nameClt] = "Bouaroua"
+      it[fstnameClt] = "Ahmed"
+      it[addressClt] = "10,Rue du Lac"
+      it[cityClt] = "Mourouj"
+      it[postalCodeClt] = 5003
+      it[ageClt] = 22
+    }
```

Comment:
maybe it would be better to call a method that adds a client like `addClient(id, name, ...)`
same thing for product and other tables.